### PR TITLE
Feature/eui 2897 wa inline messaging

### PR DESCRIPTION
**Pull request by ficklephil**
Information, Success and Warning visuals.

### CODE REVIEW

File: src/work-allocation/components/info-message/info-message.component.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+   */
+  @Input() public infoMessageType: InfoMessageType;
+
+  @Input() public infoMessage: InfoMessage;
+
+  public infoMessageType = InfoMessageType;
+
+  constructor() {}
+
+  public ngOnInit(): void {
```
Comment (paul-graham) [inline src/work-allocation/components/info-message/info-message.component.ts:22]: Is something going to go in here at some point in the future? If not, it's not needed. The same for the constructor.

**Review by connorpgpmcelroy — APPROVED** [review]

**Review by paul-graham — COMMENTED** [review]
Little bit of duplication in `info-message-component.html` due to the switch statement. Might be better to have the banner class, SVG path, and assistive message come out of the TypeScript. Everything else looks to be identical...?

File: src/work-allocation/components/info-message/info-message.component.ts (hunk 1 above)
Comment (ficklephil) [reply src/work-allocation/components/info-message/info-message.component.ts]: Good spot, left in accidentally.

**Review by paul-graham — APPROVED** [review]
All looks good to me. :)

**Pull request merged**